web/scenes/Portal/Teams/TeamId/Apps/AppId/Actions/page/CreateActionModal/server/form-schema.ts

```diff
@@ -0,0 +1,11 @@
+import * as yup from "yup";
+
+export const createActionSchema = yup.object().shape({
+  name: yup.string().required("Name is required"),
+  description: yup.string().nullable(),
+  action: yup.string().required("Action is required"),
+  app_id: yup.string().required("App ID is required"),
+  max_verifications: yup.number().required("Max verifications is required"),
```

Comment:
nit, maybe we require unsigned integers here

Reply:
ah all very good thanks!